### ISSUE TITLE
Specific post page (content page)

### ISSUE THREAD

**Issue by ShiranHi** [RedHat-Israel/RedHat-Israel.github.io#45]
Today it is hard to read the content on this page, there are two main issues:
- The font is too small
- The text line is too wide

I suggest to do some changes, the main ones are:
- Limiting the text box width to maximum 700px
- The plain text size should be 18px

Here is some more guidance:
![image](https://user-images.githubusercontent.com/77277517/204279207-b64e95ad-aa48-46d6-886c-5199edea8f62.png)

**Comment by ShiranHi** [comment]
@TomerFi in continuing with your suggestion of showing tips to users, we can surface them here on the sidebar. 
Otherwise, we can use this part to show other posts or some highlights from this post (kind of a TL;DR).

Question for all, do we want to have social icons here so users can share this specific post quickly?

**Comment by TomerFi** [comment]
Niceeee! I really like the sidebar, sound like a good fit for the tips.
@bennypowers I think something like this might also look nice with our "repos contributed to".

**Comment by bennypowers** [comment]
Yeah we could use the same data as the homepage with a smaller ui, have two sidebar sections.
